mmcv/tensorrt/__init__.py

```diff
@@ -13,7 +13,7 @@ def is_tensorrt_available():
         return False
 
 
-__all__ = []
+__all__ = []  # type: ignore
```

Comment:
What errors will be printed if not adding `#type: ignore`?

Reply:
it says "error: Type of __all__ must be "Sequence[str]", not "List[List[str]]""

Reply:
> What errors will be printed if not adding `#type: ignore`?

it says "error: Type of __all__ must be "Sequence[str]", not "List[List[str]]""

Reply:
Please update the PR by merging mmcv master branch